ClinGen/clincoded
# Connects to #736. MAF Calculations

Demo instance:
http://736-mc-calc-maf-9712c1e-minchoi.instance.clinicalgenome.org/

Images:
![image](https://cloud.githubusercontent.com/assets/4326866/16507407/75151812-3ede-11e6-9094-9d45541e3404.png)

When an interpretation is started, the CI fields come back (CI fields are not yet implemented - See #737):
![image](https://cloud.githubusercontent.com/assets/4326866/16507434/974c8a64-3ede-11e6-8da1-c6224f95e3dc.png)

Testing:

1. Create Variant with data from one or more sources (population tab) - 139214, 139215, 41231
2. Check to see that information displayed in MAF area is correct
